app/routes/govuk-components/validate-date-input-day.js

```diff
@@ -0,0 +1,58 @@
+const Joi = require('joi')
+
+const isYearEmpty = (helpers, namePrefix) => helpers.state.ancestors[0][`${namePrefix}-year`] === ''
+const isMonthEmpty = (helpers, namePrefix) => helpers.state.ancestors[0][`${namePrefix}-month`] === ''
+const isLeapYear = (year) => (year % 4 === 0 && year % 100 !== 0) || year % 400 === 0
+
+const validateDateInputDay = (namePrefix, whateverItIs) => {
```

Comment:
could you possibly change the property named 'whateverItIs' to something a bit more descriptive (similar comment for all validation files)

Reply:
<img width="598" alt="image" src="https://github.com/DEFRA/ffc-ahwr-farmer-claim/assets/118533775/51c2f96b-933b-4261-bcde-2f8a8f5eee26">

Reply:
renamed to dateName